pkg/dns/server.go

```diff
@@ -65,17 +66,20 @@ func (h *SimpleDNSServer) parseQuery(m *dns.Msg) {
 	for _, q := range m.Question {
 		switch q.Qtype {
 		case dns.TypeA, dns.TypeAAAA:
-			serverLog.V(1).Info("received a query for " + q.Name)
+			serverLog.Info("received a query", "name", q.Name, "type", q.Qtype)
```

Comment:
Do we want to switch this to `V0`? If you have thousands of Dataplanes that are using DNS, the logs would be spammed with those lines 🤔

Reply:
Arg sorry that's me debugging :( You are right I'll put it back to V(1)!